web-console/knikubevirt/expose-secret/expose-secret.md

```diff
@@ -0,0 +1,48 @@
+#Expose Secret, Config Maps and Service Accounts as Disks
+
+Kubernetes recently introduced three new volume types (Secrets, Config Maps and Service Accounts). These volumes can be used as additional VM disks, and allow better integration of Virtual Machines with Kubernetes resources.
```

Comment:
I think since this documentation will stick around for a while we can remove the reference to Kubernetes recently introducing these volume types. How about just saying "Kubernetes Secrets, Config Maps, and Service Accounts can be exposed as VM disks allowing better integration..."